openg2p_registration/models/registration_service.py

```diff
@@ -90,21 +90,25 @@ def create_disbursement_fields(self, rid, temp, regd):
                 ]:
                     org_data[k] = v
                     continue
-                if k in [
-                    "Status",
-                    "AttachmentsExpected",
-                    "AttachmentsPresent",
-                    "SubmitterName",
-                    "SubmitterID",
-                    "KEY",
-                    "meta-instanceID",
-                    "__version__",
-                    "bank_name",
-                    "city",
-                    "district",
-                    "chiefdom",
-                    "region",
-                ] or k.startswith("_"):
+                if (
+                    k
+                    in [
+                        "Status",
+                        "AttachmentsExpected",
+                        "AttachmentsPresent",
+                        "SubmitterName",
```

Comment:
These should be moved to ODK connector package.

Reply:
Yes @Ayushmina-20 is working to move the mappings to odk connector.